tensorrt_llm/_torch/modules/linear.py

```
@@ -380,9 +380,12 @@ def load_weights(self,
         if not allow_partial_loading:
             self.process_weights_after_loading(module)
 
-    def post_load_weights(self, module: Linear):
+    def transform_weights(self, module: Linear) -> None:
         pass
```

Ruff (0.15.18)

[warning] `LinearMethodBase.transform_weights` is an empty method in an abstract base class, but has no abstract decorator

(B027)